.github/workflows/PR.yml

```diff
@@ -0,0 +1,45 @@
+name: Test pull request
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
+
+jobs:
+  macos-build:
+    runs-on: macOS-latest
+
+    steps:
+    - name: Checkout the repo
+      uses: actions/checkout@v1
+    - uses: actions/cache@v1
+      with:
+        path: ~/.gradle/caches
+        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+        restore-keys: |
+          ${{ runner.os }}-gradle-
+    - name: Run emulator tests
+      uses: reactivecircus/android-emulator-runner@v2
+      with:
+        api-level: 29
+        script: ./gradlew build iosTest
```

Comment:
Shouldn't this also have connectedCheck task?

Reply:
yes